### DISCUSSION TITLE
KAKUTE F4 V1.4 FC - Acc gyro calibration not working (Betaflight works)

### DISCUSSION THREAD

**Discussion by alccrl** [iNavFlight/inav-configurator#1661] [Q&A] [answered]
Hi,
(last 6.0.0. beta inav version)
- run time calibration cross always red
- IMU not correclty detected (ICM20689 not detected, shows MPU6500)
- if ICM20689 is forced, no acc reading is performed (seems dead)
- calibration tab is always grayed, button is red

On last version 4.3.2 Betaflight, it works at first flashing (default setup).

There is some past activity in inav discussions about this type of problem, but I did not evince any conclusion about.
I can flash a custom inav firmware, if necessary, but I would appreciate some tips before, may be someone already knows exactly what the problem is, and what is the solution.

Alfredo

**Comment by alccrl** [comment, marked as answer]
With inav 5.1 and win10 configurator it installs and acc calibration works ....

So, ....
any change in code ? ..
I will have a look at it, eventually.

Alfredo

**Comment by ortas88** [comment]
Mateksys f405 se , I have the same problem
